src/tsserver/nodeServer.ts

```diff
@@ -705,7 +705,7 @@ namespace ts.server {
                     cancellationToken,
                     ...options,
                     typingsInstaller: typingsInstaller || nullTypingsInstaller,
-                    byteLength: Buffer.byteLength,
+                    byteLength: Buffer.byteLength as (buf: string, encoding?: string | undefined) => number,

```

Comment:
This would imply that everyone who calls us has to also do this sort of cast, I think. We declare our own version of this in the test runner, maybe that should be moved here instead.